common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -621,8 +652,7 @@ def available_translations(self, transcripts, verify_assets=None):
                 Defaults to `not FALLBACK_TO_ENGLISH_TRANSCRIPTS`.
 
             transcripts (dict): A dict with all transcripts and a sub.
-
-                Defaults to False
+            verify_val_transcripts(boolean): If True, verifies the edx-val transcripts as well.
```

Comment:
@muzaffaryousaf I've documented why this is being used for. It is verifying because it'll add a `lang_code` to `translations` only if it is in edx-val transcripts as well.

Reply:
but we're not verifying the val transcript but just getting them.